packages/params/src/presets/minimal.ts

```diff
@@ -115,8 +115,8 @@ export const minimalPreset: BeaconPreset = {
   // DENEB
   ///////////
   FIELD_ELEMENTS_PER_BLOB: 4096,
-  MAX_BLOB_COMMITMENTS_PER_BLOCK: 16,
-  KZG_COMMITMENT_INCLUSION_PROOF_DEPTH: 9,
+  MAX_BLOB_COMMITMENTS_PER_BLOCK: 32,
+  KZG_COMMITMENT_INCLUSION_PROOF_DEPTH: 10,
```

Comment:
It looks like there is a consensus issue with Lighthouse in our sim tests due to this, need to use a image with the updated minimal preset values.